go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/rancher
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.2
```

Comment:
Just to have a reminder for everyone. Make sure the toolchain part is not merged in. Thanks!